hyperspy/drawing/utils.py

```diff
@@ -878,135 +911,206 @@ def check_list_length(arg, arg_name):
 
     # Replot: create a list to store references to the images
     replot_ims = []
+    
+    #Below is for overlayed images
+    if overlay:
+        
+        #Check if images all have same scale and therefore can be overlayed.
+        for im in images:
+            if (im.axes_manager[0].scale != 
+                images[0].axes_manager[0].scale):
+                raise ValueError("Images are not the same scale and so should"
+                                 "not be overlayed.")
+        
+        original_cmap = plt.rcParams['image.cmap']
+        plt.rcParams['image.cmap'] = 'gray'
```

Comment:
Why is it necessary?

Reply:
This was to always set the background to black (rather than purple if the default cmap is viridis). I could add an argument to set cmap in case the user wants a different background colour. Or I could just remove and add a note on changing the default cmap if you want a different background colour.